To-do.md

```diff
@@ -0,0 +1,3 @@
+# To Do
+
+- Cargo.toml - rust-version set to 1.63.0?
```

Comment:
Is this OK?

Reply:
The Dockerfile is tied to 1.63.0 - I think I saw someplace (in `Cargo.toml`?) that it wasn't the same Rust version...

Reply:
We pin the rust version in https://github.com/prql/prql/blob/9eab4735852111ecc90341deac0952edea9ad8c4/rust-toolchain.toml#L1-L2, and it updates automatically to be n-1 behind the latest.

I think `rustup` should automatically get the later version when that changes. We could try by setting it to `1.64.0` and ensuring it still works.

(and then at a lower cadence we could update the dockerfile so it matches and doesn't have to get a new compiler version)